promtimer/promtimer.py

```diff
@@ -147,7 +170,14 @@ def prepare_grafana(grafana_port,
     make_home_dashboard()
     make_data_sources(stats_sources)
     make_dashboards_yaml()
-    make_dashboards(stats_sources, buckets, min_time_string, max_time_string, refresh)
+    if cbbackupmgr_stats_mode is False:
```

Comment:
Should this not be `if !cbbackupmgr_stats_mode`? I think `is False` checks that it is actually a boolean but I don't think you need that here - checking it is falsey is enough

Reply:
Just tested it and unfortunately, checking if it is falsey does not work :(